tfhe/src/integer/gpu/server_key/radix/scalar_div_mod.rs

```diff
@@ -32,6 +36,21 @@ impl CudaServerKey {
         result
     }
 
+    fn signed_scalar_mul_high<Scalar>(
+        &self,
+        lhs: &CudaSignedRadixCiphertext,
+        rhs: Scalar,
+        streams: &CudaStreams,
+    ) -> CudaSignedRadixCiphertext
+    where
+        Scalar: ScalarMultiplier + DecomposableInto<u8> + CastInto<u64>,
+    {
+        let num_blocks = lhs.as_ref().d_blocks.lwe_ciphertext_count().0;
+        let mut result = self.extend_radix_with_sign_msb(lhs, num_blocks, streams);
```

Comment:
Not directly related to this PR but still in the scope,`extend_radix_with_sign_msb` is not following the async/sync standards. There should exist `extend_radix_with_sign_msb_assign_async` and `extend_radix_with_sign_msb_async` so we use the latter here. The same for `signed_scalar_mul_high`, we should have an async variant and use it in `unchecked_signed_scalar_div`.

Reply:
I made the changes in this PR so everything is consistent.